modules/images/webp-uploads/load.php

```diff
@@ -90,7 +90,13 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 
 		$extension   = explode( '|', $allowed_mimes[ $targeted_mime ] );
 		$destination = trailingslashit( $original_directory ) . "{$filename}.{$extension[0]}";
-		$image       = webp_uploads_generate_additional_image_source( $attachment_id, 'full', $original_size_data, $targeted_mime, $destination );
+
+		// Skip creation of duplicate WebP image if a image file has the same name with an extension of jpe, jpg, or jpeg.
+		if ( file_exists( $destination ) ) {
+			continue;
```

Comment:
Do we want to try to create a different destination name instead? Like appending a number or a UUID instead, in the same way, WordPress handles it when the file already exists?

Reply:
@mitogh Yes, this is the eventual plan, but for now adding this stop-gap to not overwrite/confuse images should be sufficient.

Let's work on a proper solution where both images can be kept in a separate PR.